packages/components/pie-card-container/test/accessibility/pie-card-container.spec.ts

```diff
@@ -3,11 +3,31 @@ import { test, expect } from '@justeattakeaway/pie-webc-testing/src/playwright/f
 import { PieCardContainer, CardContainerProps } from '@/index';
 
 test.describe('PieCardContainer - Accessibility tests', () => {
+    test('a11y - should test the PieCardContainer component WCAG compliance if tag is "button"', async ({ makeAxeBuilder, mount }) => {
```

Comment:
Nit: could these tests have been combined?

Reply:
yea of course let me update it.